### DISCUSSION TITLE
Multi indexes for Global Search on Strapi CMS

### DISCUSSION THREAD

**Discussion by pankajnarangindia** [MattieBelt/mattie-strapi-bundle#123] [Q&A]
Hello @MattieBelt , can you please guide me whether we have implemented multiple indexes search in the plugin, so that we can implement the same in our Reactjs based UI. Seen some previous posts around combining of indexes at CMS level , but unable to get the direction on how to do that. Please help us with any post which can direct us with multi index implementation,

**Comment by praveeng-lntinfotech** [comment]
Hello @MattieBelt , can you please help me whether we have implemented multiple indexes search using Meilisearch plugin on Strapi CMS, so that we can implement the same in our React JS based UI, Please help me for multi indexes implementation,
